modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/configuration/admin/service/OpenIdConnectProviderEveryNodeEveryStartup.java

```diff
@@ -57,43 +62,13 @@
  * @review
  */
 @Component(
-	property = Constants.SERVICE_PID + "=com.liferay.portal.security.sso.openid.connect.internal.configuration.OpenIdConnectProviderConfiguration",
-	service = {
-		ManagedServiceFactory.class,
-		OpenIdConnectProviderManagedServiceFactory.class,
-		PortalInstanceLifecycleListener.class
-	}
+	configurationPid = "com.liferay.portal.security.sso.openid.connect.internal.configuration.OpenIdConnectProviderConfiguration",
+	service = PortalInstanceLifecycleListener.class
 )
 @Deprecated
-public class OpenIdConnectProviderManagedServiceFactory
+public class OpenIdConnectProviderEveryNodeEveryStartup
```

Comment:
@RenanUmV, please rename to `OpenIdConnectProviderPortalInstanceLifecycleListener`. Thx